Document fmf_url & fmf_ref metadata (#264)
packit/packit.dev: content/docs/testing-farm.md

        - fedora-all
```

Required metadata:
* **targets** - You can use `fedora-development`, `fedora-stable`, `fedora-all`
or release specific targets such as `fedora-34-x86_64`.

Optional metadata:
* **fmf_url** - Git repository containing the metadata (FMF) tree. Use any format acceptable by the git clone command.

> * **fmf_ref** - Branch, tag or commit specifying the desired git revision. Defaults to "master" if **fmf_url** is specified, but **fmf_ref** is missing.

Reviewer: ```suggestion
* **fmf_ref** - Branch, tag or commit specifying the desired git revision. Defaults to "master" when **fmf_url** is specified and **fmf_ref** is not not.
```
This is a more clear wording to me, not sure how others feel.
